src/modules/AuthServer.pm

```diff
@@ -1582,6 +1582,32 @@ sub WriteServiceSettings {
     return 1;
 }
 
+# If the system is openSUSE Tumbleweed, return '/usr/lib64/openldap'.
```

Comment:
It will also fail if I take this package and install it on any older Tumbleweed.